docs/schemas/AgInspectionReport.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$id": "https://w3id.org/traceability/schemas/AgInspectionReport.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$comment": "{\"term\": \"AgInspectionReport\", \"@id\": \"https://w3id.org/traceability/AgInspectionReport\"}",
+    "title": "Agriculture Inspection Report",
+    "description": "Information on the Inspection and the observations made",
+    "type": "object",
+    "properties": {
+      "@context": {
+        "type": "array"
+      },
+      "type": {
+          "oneOf": [
+              { "type": "string" },
+              { "type": "array"}
+          ]
+      },
+      "facilityID": {
+        "$comment": "{\"term\": \"identifier\", \"@id\": \"https://schema.org/identifier\"}",
+        "title": "Facility Identifier",
+        "description": "Identifiers for an inspection facility.",
+        "type": "string"
+      },
+      "inspectorID": {
```

Comment:
are these really strings or are the URLs? prefer URLs.

Reply:
The type says "array", I'm confused

Reply:
The array was a mistake sorry about that.  In terms of URLs...I'm not sure they would necessarily be URLs but maybe I'm not understanding.  I figured it would be an identifier that references the inspector.

Reply:
URLs pointing to how to retrieve an inspector is not a bad idea, but may not be feasible in all of our cases.  regularly we will just receive an alpha-numeric unique number that is assigned to that inspector